cmd/swarm-rafttool/dump.go

```diff
@@ -192,13 +194,15 @@ func dumpSnapshot(swarmdir, unlockKey string, redact bool) error {
 			if task != nil {
 				if container := task.Spec.GetContainer(); container != nil {
 					container.Env = []string{"ENVVARS REDACTED"}
+					container.PullOptions.RegistryAuth = "REDACTED"
```

Comment:
@dperny don't we need to check for `container.PullOptions` to be `nil`?

Reply:
good catch